### DISCUSSION TITLE
blog/2024/03/06/demo-blog-post/

### DISCUSSION THREAD

**Discussion by giscus** [bilgiyazilim/bilgiyazilim.github.io#1] [_]
# blog/2024/03/06/demo-blog-post/

http://127.0.0.1:8000/blog/2024/03/06/demo-blog-post/

**Comment by berk-karaal** [comment]
Evet Recep gerçekten de çalışıyor, istersen [şuradan](https://youtu.be/dQw4w9WgXcQ) kontrol edebilirsin :ok_hand:

**Reply by ketenburhan** [reply]
Çok doğru çalışıyor gerçekten de.
